packages/subproviders/src/subproviders/eth_lightwallet_subprovider.ts

```diff
@@ -0,0 +1,87 @@
+import { assert } from '@0xproject/assert';
+import { ECSignatureBuffer } from '@0xproject/types';
+import { addressUtils } from '@0xproject/utils';
+import * as lightwallet from 'eth-lightwallet';
+import EthereumTx = require('ethereumjs-tx');
+import * as _ from 'lodash';
+
+import { PartialTxParams, WalletSubproviderErrors } from '../types';
+
+import { BaseWalletSubprovider } from './base_wallet_subprovider';
+
+/*
+ * This class implements the web3-provider-engine subprovider interface and forwards
+ * requests involving user accounts and signing operations to eth-lightwallet
+ *
+ * Source: https://github.com/MetaMask/provider-engine/blob/master/subproviders/subprovider.js
+ */
```

Comment:
Can you add a quick description for this subprovider, and also the link to the EthLightwallet repo?